lib/spack/spack/concretize.py

```diff
@@ -83,6 +89,140 @@ def concretize_specs_together(*abstract_specs, **kwargs):
     return [s.copy() for s in result.specs]
 
 
+def concretize_together(*spec_list, **kwargs):
```

Comment:
Just wanted to note that in this file we added `**kwargs` and no type-hinting to new functions.